packages/fluentui/react-northstar/src/themes/teams/components/Carousel/carouselVariables.ts

```diff
@@ -46,29 +46,28 @@ export const carouselVariables = (siteVars): CarouselVariables => ({
   focusOuterBorderColor: siteVars.focusOuterBorderColor,
   focusOuterBorderRadius: siteVars.borderRadius,
 
-  paddleHeight: pxToRem(32),
-
-  paddleColor: siteVars.colorScheme.default.foreground,
-  paddleBackgroundColor: siteVars.colorScheme.default.background,
-  paddleBorderRadius: siteVars.borderRadius,
-  paddleBorderColor: siteVars.colorScheme.default.border,
-  paddleBoxShadow: siteVars.shadowLevel1,
-
-  paddleColorHover: siteVars.colorScheme.brand.foregroundHover,
-  paddleBackgroundColorHover: siteVars.colorScheme.default.backgroundHover2,
-  paddleBorderColorHover: siteVars.colorScheme.default.borderHover,
-
-  paddleColorActive: siteVars.colorScheme.default.foregroundPressed,
-  paddleBackgroundColorActive: siteVars.colorScheme.default.backgroundPressed,
-  paddleBorderColorActive: siteVars.colorScheme.default.borderPressed,
+  paddleColor: siteVars.colorScheme.default.foreground4,
+  paddleColorActive: undefined,
+  paddleColorHover: undefined,
+  paddleColorFocus: undefined,
```

Comment:
what is the motivation of having undefined variables ?

Reply:
good point, removing